src/app/routes/endpoints/events/initial.js

```diff
@@ -74,7 +74,7 @@ const message = {
+        "\n\n:smile:  Don't be shy! Introduce yourself in <https://codebuddies.slack.com/archives/C04AQ6GH4|#introduce-yourself> &amp; welcome fellow new members. _The more we know about each other, the better we can connect &amp; collaborate._\n\n\n:point_left::skin-tone-5:  Browse all our public Slack channels by clicking on `CHANNELS` over in the sidebar.\n\n\n:eyeglasses:  Make sure to read our <https://codebuddies.org./slack Etiquette for General Members>.\n\n\n:computer:  Log into our <http://codebuddies.org|website> using your new CodeBuddies Slack account &amp; take a look around -- or join a hangout.\n\n\n:calendar: Try scheduling a hangout yourself! (_You don’t need to be an expert to collaborate._) Simply click on the `Schedule a Hangout` button on the website &amp; fill in the information. All hangouts get automatically posted back to <https://codebuddies.slack.com/archives/C04AQ6GH0|#announcements>.\n\n\n:books:  Interested in going deep on topics like * Functional JS* or that special *Ruby* book or *Blockchain*? Start a <codebuddies.org/study-groups|StudyGroup>!  All study groups created on the website also get automatically posted back to <https://codebuddies.slack.com/archives/C04AQ6GH0|#announcements>.  Invite folks here on Slack to join in so that you can motivate each other to master your learning goal.\n\n\n:question:  If you have any questions about the community or the project or want to talk to an admin, feel free to ask about it in <https://codebuddies.slack.com/archives/C04BRN86J|#codebuddies-meta>. It's where discussions related to our community happen.\n\n\n:heavy_plus_sign:  If you'd like to add a new integration to the Slack, please ask in the <https://codebuddies.slack.com/archives/C04BRN86J|#codebuddies-meta> channel as well! Because CodeBuddies is on a free plan, the number of new Slack apps we can integrate is limited, so we would need to choose our integrations wisely.\n\n\n_Happy Learning!_ :wave::skin-tone-5:",
```

Comment:
Looks like a stray dot there: `https://codebuddies.org./slack`

Reply:
and the pipe is missing between link and text

Reply:
Fixed, thank for spotting it out